js/script.js

```diff
@@ -0,0 +1,48 @@
+$(document).ready(function() {
+
+var searchResults = [];
+var imageArray = [];
+var testImage;
+var counter = 0;
+var searchTerm;
+
```

Comment:
Let's hide the reset button on document load, since we don't need it to display until after the slide show starts:

```
	$('#reset-button').hide();
```